### ISSUE TITLE
Review 'cell proliferation' branch

### ISSUE THREAD

**Issue by paolaroncaglia** [geneontology/go-ontology#12229]
As discussed at an ontology call:

"Most ['cell proliferation' terms] have logical def, e.g. 'epithelial cell proliferation' is_a 'cell proliferation' that acts_on_population_of 'epithelial cell'. Did we have plans for a TG template? (Not in Jira.)
Note: this is a case where we have a semi-random distribution of terms in the ontology vs annotation extensions. Should we just make terms for all AE usage and retrofit AEs?

 Should we have a template?
 Yes. Paola will request it. 
DONE: https://github.com/geneontology/termgenie/issues/85

 Many are in AEs - should all those go into the ontology?
 Melanie:  Isn't this going to cause bloat?  Shouldn't we be pushing in the other direction?
 David: Doing this would hide the annotation from term enrichment (only inferred parent classes would be visible)
 Paola: Enrichment to such specific terms would be rare.

[THIS IS THE OBJECT OF THIS TICKET:]
Decision: once the template is ready, create terms for children of 'cell proliferation' that have abundant annotation extensions. 
 Then look at the structure under 'cell proliferation': if it's too flat, see if we can create some intermediate terms (using the template).
 Paola can do it (not as a top priority though)."

(For reference, minutes here:
http://wiki.geneontology.org/index.php/Ontology_meeting_2015-10-29#.27cell_proliferation.27_terms)

**Comment by paolaroncaglia** [comment]
I think this is really low priority, and didn't stem from a curator's request, so I'll close it. The idea could be revisited in the future if there is interest and resources, but the same might apply for other branches in GO that have annotation extensions.
